python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -568,8 +513,8 @@ def test_dataframe_dir_and_getattr():
     assert "oop$" not in o
 
     # Getattr works
-    assert df.a is df["a"]
-    assert df.b is df["b"]
+    assert df.a.equals(df["a"])
```

Comment:
I guess these return different Series objects now represented by the same underlying memory?

Reply:
Right